AutomatedTesting/Gem/PythonTests/Prefab/tests/duplicate_prefab/DuplicateEntity_WithNestedEntities.py

```diff
@@ -0,0 +1,65 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+def DuplicateEntity_WithNestedEntities():
+    """
+    Test description:
+    - Creates linear nested entities.
+    - Duplicates the nested entity structure, and validates.
```

Comment:
If it duplicates the entity structure, does it keep the string name the same, but the UUIDs different?

Reply:
Yes, the entity name will be duplicated, but the duplicated entities will have new entityIds.